pkg/controller/controller_broker.go

```diff
@@ -363,14 +430,62 @@ func (c *controller) reconcileServiceClassFromServiceBrokerCatalog(broker *v1alp
 
 	toUpdate := clone.(*v1alpha1.ServiceClass)
 	toUpdate.Bindable = serviceClass.Bindable
-	toUpdate.Plans = serviceClass.Plans
 	toUpdate.PlanUpdatable = serviceClass.PlanUpdatable
 	toUpdate.Tags = serviceClass.Tags
 	toUpdate.Description = serviceClass.Description
 	toUpdate.Requires = serviceClass.Requires
 
 	if _, err := c.serviceCatalogClient.ServiceClasses().Update(toUpdate); err != nil {
-		glog.Errorf("Error updating serviceClass %v from ServiceBroker %v: %v", serviceClass.Name, broker.Name, err)
+		glog.Errorf("Error updating serviceClass %q from ServiceBroker %q: %v", serviceClass.Name, broker.Name, err)
```

Comment:
nit: it would be cool to fix the capitalization issues here

Reply:
sure why not.